workflow/controller/config.go

```diff
@@ -22,11 +22,16 @@ import (
 // WorkflowControllerConfig contain the configuration settings for the workflow controller
 type WorkflowControllerConfig struct {
 	// ExecutorImage is the image name of the executor to use when running pods
+	// DEPRECATED: use --executor-image flag to workflow-controller instead
```

Comment:
Do you mean --executor instead of --executor-image?

Reply:
The flag is actually `--executor-image` in the CLI:

```golang
	command.Flags().StringVar(&executorImage, "executor-image", "", "Executor image to use (overrides value in configmap)")
```